src/MOI/MOI_wrapper.jl

```diff
@@ -2801,21 +2818,34 @@ function MOI.get(model::Optimizer, attr::MOI.DualStatus)
     end
 end
 
+_update_cache(::Optimizer, data::Vector{Float64}, ::Any) = data
+
+function _update_cache(model::Optimizer, ::Nothing, f_p::F) where {F}
+    n = length(model.variable_info)
+    x = zeros(n)
+    ret = f_p(model.env, model.lp, x, 0, n-1)
```

Comment:
Is the parameter `f_p` really required? You're only using `CPXgetx` and it makes the call look strange.

Reply:
Agreed. Please hard-code `CPXgetx`. We can refactor in future if needed.

Reply:
yes I was thinking maybe we would need something similar for the dual as well, but for now it's ok to hard-code, it's done.